Installing-and-Configuring/Flashing-Guides/T480-maximized.md

```diff
@@ -0,0 +1,128 @@
+Please note, at the moment (Feb 2025) the Thunderbolt data transfer is not supported upstream by coreboot. However, the video output through the Thunderbolt and charging works. So, only the usb-c charging port can be used for the data transfer. Heads provides fixed and padded Thunderbolt firmware for your convenience that will only fix "charging problem" if the bug affected your laptop. The flashing steps are based on libreboot strategy and described in the second part of the guide. Board testers did not have the problem. It is unlikely that you get it if your laptop was in use longer than 12 months before. If you get "charging bug" it is possible to fix with external flashing. However, please note that this neither was tested after Heads installation nor the bug appeared after extensive Heads testing of the board.
+
+## Flashing Heads
+
+First remove the battery and the cable powering your device. Removing these screws will allow you to remove the back panel. Using a guitar pick or an old credit card may be helpful for removing the panel.
+
+![Back view]({{ site.baseurl }}/images/T480/1_screws.jpg)
+
+The back panel and the battery are removed. Important, detach the internal battery and CMOS. Arrows point to the direction you should pull the connectors. Pull the plastic part and not the wires, since they are thin and can be damaged.
+
+![Battery]({{ site.baseurl }}/images/T480/2_battery.jpg)
+
+Top right chip corresponds to the Thunderbolt SPI flash chip (1 mb). Chip located in the middle of the board corresponds to the BIOS (16 mb) chip, respectively. 
+
+The chip located in the middle of the board contains the [Intel Management Engine (ME)](https://www.flashrom.org/ME) firmware.
+
+![Chips]({{ site.baseurl }}/images/T480/3_chips.jpg)
+
+First [download]({{ site.baseurl }}/Downloading)  or build (please see [general building]({{ site.baseurl }}/{{ site.baseurl }}/x230-maximized-building/) / [building x230]({{ site.baseurl }}/x230-maximized-building/)) the board rom for this board and verify its hash value.
+
+
+Try to read the name of the SPI flash chip. The dot on the chip helps to identify the correct clip orientation. 
+
+![SPI BIOS flash chip closed view]({{ site.baseurl }}/images/T480/4_bios_chip_orientation.jpg)
+
+ First, connect the clip of the ch341a programmer to the chip. Next, connect the programmer to the usb port of your other Linux-based computer with flashrom installed. In my set up, the red wire should be where the dot is (dot indicates pin 1). Here, please see flashing guide for the t430. Using raspberry pi pico is nicely described in the libreboot guide [Libreboot flash ](https://libreboot.org/docs/install/spi.html).
```

Comment:
I woukd advise for once to own a ch341a revision 1.6+ (1.6, 1.7..) because of voltage regulator present and enforced properly.